test/unittests/util/test_format.py

```diff
@@ -429,17 +441,19 @@ def test_nice_year(self):
                 self.assertEqual(p['assertEqual'], nice_year(
                     dt, lang=lang, bc=ast.literal_eval(p['bc'])))
                 i = i + 1
+        set_default_lf_lang(default_lang)
 
         # Test all years from 0 to 9999 for all languages,
         # that some output is produced
         for lang in self.test_config:
+            set_default_lf_lang(lang)
```

Comment:
This may or may not fail on some runs. Neither unittest nor pytest run tests in a reliable order. However, if it succeeds, this beats the alternative, which is spamming a function that unloads all the other languages.

A third option is setting `config.load_langs_on_demand`, but this is different behavior from what Mycroft users will usually see at runtime, so not ideal for these tests.

Reply:
So far haven't seen any issues but good to note in case that randomly starts ocurring and we can't work out why